deps/curl-7.58.0/docs/curl.1

```diff
@@ -22,8 +22,7 @@
 .\"
 .\" DO NOT EDIT. Generated by the curl project gen.pl man page generator.
 .\"
-.TH curl 1 "November 16, 2016" "Curl 7.58.0" "Curl Manual"
-
+.TH curl 1 "16 Dec 2016" "Curl 7.52.0" "Curl Manual"
```

Comment:
This is an interesting update. Did you have 7.52.0. somewhere on your machine?

Reply:
I am not sure. My machine has libcurl 7.54.0. I have no idea where this comes from.